packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -236,6 +243,93 @@ - (instancetype)initWithPlayerItem:(AVPlayerItem *)item
   return self;
 }
 
+- (void)setPictureInPicture:(BOOL)pictureInPicture {
+    if (self._pictureInPicture == pictureInPicture) {
+        return;
+    }
+
+    self._pictureInPicture = pictureInPicture;
```

Comment:
if the value is the same, should we do early return?

Reply:
Isn't that what I am doing on line 247 & 248?